Framework/PythonInterface/test/python/plugins/algorithms/AbinsBasicTest.py

```diff
@@ -200,6 +211,37 @@ def test_partial(self):
                                                    Tolerance=self._tolerance)
             self.assertEqual(result, True)
 
+    def test_numbered(self):
+        # Can request workspaces by atom number; these should add up to the usual elemental totals
+
+        wrk_ref = Abins(AbInitioProgram=self._ab_initio_program,
+                        VibrationalOrPhononFile=self._squaricn + ".phonon",
+                        Atoms=self._atoms,
+                        QuantumOrderEventsNumber=self._quantum_order_events_number,
+                        ScaleByCrossSection=self._cross_section_factor,
+                        OutputWorkspace=self._squaricn + "_ref")
+
+        numbered_workspace_name = "numbered"
+        h_indices = ("1", "2", "3", "4")
+        wks_numbered_atoms = Abins(VibrationalOrPhononFile=self._squaricn + ".phonon",
+                                   Atoms=", ".join([AbinsConstants.ATOM_PREFIX + s for s in h_indices]),
+                                   SumContributions=self._sum_contributions,
+                                   QuantumOrderEventsNumber=self._quantum_order_events_number,
+                                   ScaleByCrossSection=self._cross_section_factor,
+                                   OutputWorkspace=numbered_workspace_name)
+
+        wrk_ref_names = list(wrk_ref.getNames())
```

Comment:
`getNames()` might be deprecated. Or I could be thinking of `getName()`, I'm not sure 🤔

Reply:
`wrk.getName()` raises a deprecation warning, suggesting `wrk.name()` instead.  `wrk.getNames()` raises no such warning and I don't see an alternative method.

Both getName() and getNames() are suggested in the WorkspaceGroup Python docs [here](https://docs.mantidproject.org/nightly/concepts/WorkspaceGroup.html) . name() is used [here](https://www.mantidproject.org/Python_Managing_Workspaces) but that's a little confusing because it appears in a paragraph titled "Deprecated way".